gammapy/modeling/parameter.py

```diff
@@ -97,6 +97,8 @@ class Parameter:
         Whether the parameter represents the flux norm of the model.
     prior : `~gammapy.modeling.models.Prior`
         Prior set on the parameter.
+    scale_interp : {"lin", "sqrt", "log"}
```

Comment:
Can you profit of this PR to add in the docstring all the default values? Thanks